src/app/modules/main/chat_section/module.nim

```diff
@@ -929,7 +929,9 @@ method onNewMessagesReceived*(self: Module, sectionIdMsgBelongsTo: string, chatI
   let contactDetails = self.controller.getContactDetails(message.`from`)
   let communityChats = self.controller.getCommunityById(chatDetails.communityId).chats
   let renderedMessageText = self.controller.getRenderedText(message.parsedText, communityChats)
-  let plainText = singletonInstance.utils.plainText(renderedMessageText)
+  var plainText = singletonInstance.utils.plainText(renderedMessageText)
+  if ContentType(message.contentType) == ContentType.Image and len(plainText) == 0:
+    plainText = "Image"
```

Comment:
:police_officer: `qsTr()` needed since it's a user visible string... oh wait :)

You need to defer the actual translation to QML

Reply:
hm... but what about OS notifications? :)

Reply:
Yeah, mark the string as a Constant, and then translate it in QML (see `Constants.qml, appTranslatableConstants enum`)

Reply:
No, we don't have a way to do this in `nim`

Reply:
I think its possible but not comfortable :) or [this may help](https://github.com/juancarlospaco/nim-nimterlingua).
I you asking about QML translation, can we resolve the conversation cos this string displayed only in OS Notification?

Reply:
I still think it's doable using the constants, wdyt @saledjenic ?

Reply:
I do remember that for the image notification, we send an image emoji instead of the text

Reply:
![image](https://user-images.githubusercontent.com/117639195/231180772-edb495a5-9ee4-4487-983e-ea5638ab14b0.png)

Reply:
Why not simply use the "image" emoji here (just for the notification text) then? It's technically still a text and will be correctly rendered by the OS notification